pkg/flowaggregator/exporter/clickhouse.go

```diff
@@ -31,21 +33,36 @@ type ClickHouseExporter struct {
 	chExportProcess *clickhouseclient.ClickHouseExportProcess
 }
 
+const (
+	CACertFile = "ca.crt"
+	CertDir    = "/etc/ssl/certs"
+)
+
 func buildClickHouseConfig(opt *options.Options) clickhouseclient.ClickHouseConfig {
 	return clickhouseclient.ClickHouseConfig{
-		Username:       os.Getenv("CH_USERNAME"),
-		Password:       os.Getenv("CH_PASSWORD"),
-		Database:       opt.Config.ClickHouse.Database,
-		DatabaseURL:    opt.Config.ClickHouse.DatabaseURL,
-		Debug:          opt.Config.ClickHouse.Debug,
-		Compress:       opt.Config.ClickHouse.Compress,
-		CommitInterval: opt.ClickHouseCommitInterval,
+		Username:           os.Getenv("CH_USERNAME"),
+		Password:           os.Getenv("CH_PASSWORD"),
+		Database:           opt.Config.ClickHouse.Database,
+		DatabaseURL:        opt.Config.ClickHouse.DatabaseURL,
+		Debug:              opt.Config.ClickHouse.Debug,
+		Compress:           opt.Config.ClickHouse.Compress,
+		CommitInterval:     opt.ClickHouseCommitInterval,
+		CustomCACert:       opt.Config.ClickHouse.TLS.CustomCACert,
+		InsecureSkipVerify: opt.Config.ClickHouse.TLS.InsecureSkipVerify,
 	}
 }
 
 func NewClickHouseExporter(k8sClient kubernetes.Interface, opt *options.Options) (*ClickHouseExporter, error) {
 	chConfig := buildClickHouseConfig(opt)
-	klog.InfoS("ClickHouse configuration", "database", chConfig.Database, "databaseURL", chConfig.DatabaseURL, "debug", chConfig.Debug, "compress", *chConfig.Compress, "commitInterval", chConfig.CommitInterval)
+	klog.InfoS("ClickHouse configuration", "database", chConfig.Database, "databaseURL", chConfig.DatabaseURL, "debug", chConfig.Debug, "compress", *chConfig.Compress, "commitInterval", chConfig.CommitInterval, "customCACert", chConfig.CustomCACert)
```

Comment:
Would we also like to log InsecureSkipVerify here?

Reply:
Added